StreamsH/src/com/sci/bv/Load.java

```diff
@@ -0,0 +1,62 @@
+package com.sci.bv;
+
+
+
+public class Load implements Comparable<Load>{
```

Comment:
so you can use this object to load your values from a file
it will be able to sort/compare
it will detect the proper month of birth

but you don't have a main class/method to call this
you're not reading from a file, you're not writing to a file, you're not actually sorting or filtering anything